htdocs/expedition/card.php

```diff
@@ -1526,6 +1526,12 @@
 										} else {
 											print 'TableLotIncompleteRunRepairWithParamStandardEqualConfirmed';
 										}
+										if (empty($conf->global->PRODUCT_DISABLE_SELLBY)) {
+											print ' - '.$langs->trans("SellByDate").': '.dol_print_date($dbatch->sellby, "day");
```

Comment:
Sometime the sellby and eatby date are used only for few products and others just use a lot.
Is showing the sellby and eatby title always for each lot even if value is empty required ?
What do you think of adding a test 
`& !empty($dbatch->sellby)`

Reply:
@eldy 
You're right it will be better, so I updated this PR